code/modules/mob/mob_movement.dm

```diff
@@ -26,25 +26,6 @@
 	else
 		to_chat(usr, "<span class='danger'>This mob type cannot throw items.</span>")
 
-
-/client/verb/drop_item()
```

Comment:
Will remove the `drop-item` verb for people to use while typing in the command bar.
Dunno if anybody uses that though but it's a removal that is not described in the PR